### DISCUSSION TITLE
can Quasar CLI (with Vite)  provide `htmlFilename` option?

### DISCUSSION THREAD

**Discussion by taisha** [quasarframework/quasar#13499] [General - Components / Directives / etc] [answered]
The entry file is always `index.html`  after `quasar build `  (with Vite). 
I try to find a way to produce arbitrary html file name. Just like the way `htmlFilename` option in Quasar CLI (with Webpack)  does.

Any one could give me some advice?
Thank you!

**Comment by rstoenescu** [comment, marked as answer]
Not yet possible due to how Vite works. Its expectations are that it will look for index.html, hence I had to move /src/index.template.html to /index.html for q/app-vite projects.
